src/main/java/AddCommand.java

```diff
@@ -15,6 +15,13 @@ public String toString() {
 
     @Override
     public void execute() {
+        User user = this.createNewUser();
+        usersBook.addUser(user);
+        System.out.println("New user created:\n" + user + "\nUser ID: " + user.getId());
+        System.out.println(SEPARATOR);
+    }
+
+    public User createNewUser() {
```

Comment:
you don't need it.  You can extend a bit user book and add method to find last added user.
AddUserCommand.execute(); 
User user = userBook.getLastAddedUser()